forms/serializers/form.py

```diff
@@ -535,46 +535,39 @@ class Meta:
 
     @staticmethod
     def get_applicant(obj):
+        applicant_list = list()
         try:
             applicant_section = obj.entry_sections.get(
```

Comment:
I realized that I didn't actually have an application with 2+ applicants yet (probably because I remember having trouble with that before), so I created one and got a 500 with the following trace:

```
mvj         | 172.18.0.1 - - [18/Aug/2022 13:54:25] "GET /v1/answer/?limit=25 HTTP/1.1" 500 -
mvj         | Traceback (most recent call last):
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py", line 76, in __call__
mvj         |     return self.application(environ, start_response)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/wsgi.py", line 133, in __call__
mvj         |     response = self.get_response(request)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 130, in get_response
mvj         |     response = self._middleware_chain(request)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner
mvj         |     response = response_for_exception(request, exc)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception
mvj         |     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 149, in handle_uncaught_exception
mvj         |     return debug.technical_500_response(request, *exc_info)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django_extensions/management/technical_response.py", line 37, in null_technical_500_response
mvj         |     six.reraise(exc_type, exc_value, tb)
mvj         |   File "/usr/local/lib/python3.8/site-packages/six.py", line 702, in reraise
mvj         |     raise value.with_traceback(tb)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
mvj         |     response = get_response(request)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
mvj         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
mvj         |     return view_func(*args, **kwargs)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/viewsets.py", line 114, in view
mvj         |     return self.dispatch(request, *args, **kwargs)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 505, in dispatch
mvj         |     response = self.handle_exception(exc)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 465, in handle_exception
mvj         |     self.raise_uncaught_exception(exc)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
mvj         |     raise exc
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 502, in dispatch
mvj         |     response = handler(request, *args, **kwargs)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/mixins.py", line 43, in list
mvj         |     return self.get_paginated_response(serializer.data)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 760, in data
mvj         |     ret = super().data
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 260, in data
mvj         |     self._data = self.to_representation(self.instance)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 677, in to_representation
mvj         |     return [
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 678, in <listcomp>
mvj         |     self.child.to_representation(item) for item in iterable
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 529, in to_representation
mvj         |     ret[field.field_name] = field.to_representation(attribute)
mvj         |   File "/usr/local/lib/python3.8/site-packages/rest_framework/fields.py", line 1905, in to_representation
mvj         |     return method(value)
mvj         |   File "/code/forms/serializers/form.py", line 540, in get_applicant
mvj         |     applicant_section = obj.entry_sections.get(
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
mvj         |     return getattr(self.get_queryset(), name)(*args, **kwargs)
mvj         |   File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 439, in get
mvj         |     raise self.model.MultipleObjectsReturned(
mvj         | forms.models.form.EntrySection.MultipleObjectsReturned: get() returned more than one EntrySection -- it returned 2!
```

Reply:
The crash is now gone, though using `.first()` implicitly cuts out all but the first found applicant, which kind of defeats the point. The applicant test should be done to every EntrySection.

Reply:
I noticed that as well. I think that this is first time that I want .first() to give exception